src/Octokit.Webhooks.AspNetCore/GitHubWebhookExtensions.cs

```diff
@@ -106,7 +120,7 @@ await context.Response.WriteAsync("Payload includes a secret, so the webhook rec
             return false;
         }
 
-        var keyBytes = Encoding.UTF8.GetBytes(secret);
+        var keyBytes = Encoding.UTF8.GetBytes(secret!);
```

Comment:
Should this default to an empty string, or error out if secret is null or empty?

Reply:
Since the secret is optional, should it just `return true;` if its `null`?

Reply:
It's a little hard to follow, but this is covered with the logic above.

If the secret is `null` or empty then `isSignatureExpected` will be `false`. We cover 3 of the combinations of `isSigned` and `isSignatureExpected` above. By the time we get here we know the event is signed, and we know we are expecting a signature.